services/storage/src/simcore_service_storage/models.py

```diff
@@ -151,6 +151,7 @@ class FilesMetadataDatasetQueryParams(StorageQueryParamsBase):
 
 
 class FilesMetadataQueryParams(StorageQueryParamsBase):
+    project_id: ProjectID | None = None
```

Comment:
this already sets a default right? so the api should not have it correct?

Reply:
Yep, I removed them in the api, thanks